File: Python/lista2 - estruturas de repeticao/exercicio_10.py
i = 1
number = 0
total = 0
positive = 0;
negative = 0

while i != 0:
    number = float(input('Informe um número: '))

    if number > 0:
        positive += 1
    elif number < 0:
        negative += 1

    if number == 0:
        break

    total += number
    i += 1

print(f'Média dos valores: {total / (i - 1)}')
print(f'Total de valores positivos: {positive}')
print(f'Total de valores Negativo: {negative}')
print(f'Percentual de valores positivos: {(positive / (i - 1) * 100)}%')
print(f'Percentual de valores negativos: {(negative / (i - 1) * 100)}%')

File: C/lista2 - estrutura de repeticao/exercicio_10.c
#include <conio.h>
#include <stdio.h>

int main() {
	int addNumbers = 0, i = 1, negative = 0, number, positive = 0;
	float average = 0;
	
	for (i= 0; i!= 0; i++) {
		printf("Informe um numero: ");
		scanf("%d", &number);
		
		if (number >= 0 && number != 0) {
			positive++;
		} else {
			negative++;
		}
		
		if (number == 0) {
			break;
		}
		
		addNumber += number;
	}
	
	average = (float) addNumbers / (i);
	
	printf("Media dos valores: %f\n", average);
	printf("Total de valores positivos: %d\n", positive);
	printf("Total de valores negativos: %d\n", negative);
	printf("Percentual de valores positivos: %f\n", (float) (positive / (i) * 100));
	printf("Percentual de valores negativos: %f\n", (float) (negative / (i) * 100));
}